### PR TITLE
Guard tests with REQUIRES:aarch64-registered-target

### PR DESCRIPTION
**Pull request by rastogishubham**

### CODE REVIEW

**Comment by rastogishubham** [comment]
@swift-ci please test llvm

**Comment by rastogishubham** [comment]
@swift-ci please test llvm

File: llvm/test/DebugInfo/AArch64/swift-async.ll [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ; RUN: llc -O0 -filetype=obj < %s | llvm-dwarfdump --name n - | FileCheck %s
 ;
+;
```
Comment (cachemeifyoucan) [inline llvm/test/DebugInfo/AArch64/swift-async.ll]: Avoid whitespace change

File: llvm/test/DebugInfo/CAS/AArch64/debug_unopt.ll [hunk 2]
```diff
@@ -6,6 +6,7 @@
 ; CHECK: mc:debug_line_section llvmcas://
 ; CHECK-NEXT: mc:debug_line_unopt llvmcas://
 ; CHECK-NEXT: mc:padding      llvmcas://
+
```
Comment (cachemeifyoucan) [inline llvm/test/DebugInfo/CAS/AArch64/debug_unopt.ll]: same here.

File: llvm/test/DebugInfo/CAS/AArch64/multiple_cus.ll [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ; RUN: llc -cas-friendly-debug-info -O0 --filetype=obj --cas-backend --cas=%t/cas --mccas-verify --mtriple=arm64-apple-darwin %s -o %t/multiple_cus.o 2>&1 | FileCheck %s --allow-empty
 ; CHECK-NOT: error in backend: CASBackend output round-trip verification error
 
+; REQUIRES: arm64
```
Comment (cachemeifyoucan) [inline llvm/test/DebugInfo/CAS/AArch64/multiple_cus.ll]: Drop this.

**Review by cachemeifyoucan — COMMENTED** [review]
Also fix the commit message. Otherwise, LGTM.

**Review by cachemeifyoucan — APPROVED** [review]

**Comment by rastogishubham** [comment]
@swift-ci please test llvm

**Pull request merged**